src/app/my-info/(account)/orders/_components/Orders/Orders.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { useQuery } from '@tanstack/react-query';
+
+import { getOrdersData } from '@/api/orderAPI';
+import { MyInfoEmptyCase } from '@/app/my-info/_components';
+import DatePicker from '@/components/DatePicker/DatePicker';
+import { formatStartDate } from '@/libs/formatStartDate';
+import type { Order } from '@/types/orderType';
+import { useEffect, useState } from 'react';
+import { OrderHeader, OrderItemList } from './index';
+
+export default function Orders() {
+  const { data: orders } = useQuery<{ data: Order[] }>({ queryKey: ['ordersData'], queryFn: getOrdersData });
+
+  const [searchDate, setSearchDate] = useState('');
+  const [ordersData, setOrdersData] = useState(orders?.data ?? []);
+
+  useEffect(() => {
+    if (orders && searchDate !== '') {
+      const filteredOrders = orders.data.filter((order) => new Date(order.purchaseDate) > new Date(searchDate));
+      setOrdersData(filteredOrders);
+    }
+  }, [searchDate, orders]);
+
+  const handleDateClick = (date: { startDate: Date; endDate: Date }) => {
+    const startDate = new Date(date.startDate);
+    setSearchDate(formatStartDate(startDate));
+  };
```

Comment:
```suggestion
  const handleDateClick = (date: { startDate: Date; endDate: Date }) => {
    setSearchDate(formatStartDate(date.startDate));
  };
```

혹시 이렇게 해도 되지 않을까요? 아니면 명확하게 보여주고 싶으셔서???